python/tk_multi_workfiles/file_list/user_filter_menu.py

```diff
@@ -120,8 +122,8 @@ def _set_selected_users(self, users):
     def _get_available_users(self):
         available_users = set(
             [
-                details.user
-                for details in self._available_users.values()
+                user_id
```

Comment:
This method was not called anywhere in the code base and when I tried to invoke it in my tests it failed because `dict` is not a hashable type, so I fixed the method the same way we implemented selected_users_ids.